src/Resources/maintenance/index.html

```diff
@@ -1,9 +1,24 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title></title>
+        <title>In maintenance</title>
```

Comment:
:smile: Maybe just the same as within the body: `We're currently upgrading our systems...`